remote-api/src/test/java/org/alfresco/rest/api/impl/CategoriesImplTest.java

```diff
@@ -751,6 +831,31 @@ public void testUpdateCategoryById_includeCount()
             .isEqualTo(1);
     }
 
+    @Test
+    public void testUpdateCategoryById_includePath()
+    {
+        final String categoryNewName = "categoryNewName";
+        final Category fixedCategory = createCategoryOnlyWithName(categoryNewName);
+        fixedCategory.setPath("/test/TestCat");
```

Comment:
[Minor] Here I think we're testing that if the client provides a path then it is ignored. It might be nice to mention that in a comment.

Reply:
You're right, noted thanks!